README.markdown

```diff
@@ -51,12 +51,13 @@ http {
         local ok, err = hc.spawn_checker{
             shm = "healthcheck",  -- defined by "lua_shared_dict"
             upstream = "foo.com", -- defined by "upstream"
-            type = "http",
-
+            
+            type = "https",
```

Comment:
Because `type = "http"` is a more common case, we should at least put it in a comment here?